packages/scandipwa/src/component/ProductAttributeValue/ProductAttributeValue.component.js

```diff
@@ -307,15 +307,15 @@ export class ProductAttributeValue extends PureComponent {
     }
 
     renderStringValue(value, label, count) {
-        const { isFormattedAsText, isSelected } = this.props;
+        const { isFormattedAsText, isSelected, isProductCountVisible } = this.props;
         const isSwatch = label;
 
         if (isFormattedAsText) {
             return label || value || __('N/A');
         }
 
         if (!isSwatch) {
-            return this.renderDropdown(value, count);
+            return this.renderDropdown(value, isProductCountVisible ? count : null);
```

Comment:
Tho, you can actually just check it in function that I written about above and if isProductCountVisible set to false just not render subLabel for checkboxLabel.